include/seqan3/io/structure_file/format_vienna.hpp

```diff
@@ -129,9 +129,9 @@ class structure_file_format_vienna
               comment_type & SEQAN3_DOXYGEN_ONLY(comment),
               offset_type & SEQAN3_DOXYGEN_ONLY(offset))
     {
-        auto stream_view = view::subrange<decltype(std::istreambuf_iterator<char>{stream}),
-                                          decltype(std::istreambuf_iterator<char>{})>
-                           { std::istreambuf_iterator<char>{stream}, std::istreambuf_iterator<char>{} };
+        using stream_it_t = std::istreambuf_iterator<typename stream_type::char_type>;
+        auto stream_view = view::subrange<decltype(stream_it_t{stream}), decltype(stream_it_t{})>{stream_it_t{stream},
```

Comment:
instead of writing `decltype(stream_it_t{stream})` you can just write `stream_it_t` :smile:

Reply:
okay!